glib-sys/src/lib.rs

```diff
@@ -34,6 +34,12 @@ pub type GCallback = extern "C" fn();
 pub type GClosureNotify = extern "C" fn(data: gpointer, closure: gpointer);
 
 #[repr(C)]
+pub struct C_GValue {
+    typ: GType,
```

Comment:
Can make this `type_` I guess. Prefixing underscores has a meaning in Rust so I'm against that.